dm/dm/master/server.go

```diff
@@ -37,6 +39,7 @@ import (
 	"google.golang.org/grpc"
 
 	"github.com/pingcap/tiflow/dm/checker"
+	"github.com/pingcap/tiflow/dm/dm/common"
```

Comment:
already have a import call `dmcomman` in L43 😂, could you pelase help to merge this two import?